src/server/game/Movement/MotionMaster.cpp

```diff
@@ -387,6 +387,43 @@ void MotionMaster::MoveJump(float x, float y, float z, float speedXY, float spee
     Mutate(new EffectMovementGenerator(id), MOTION_SLOT_CONTROLLED);
 }
 
+void MotionMaster::MoveCirclePath(float x, float y, float z, float radius, bool clockwise)
+{
+    float step = clockwise ? float(-M_PI) / 8.0f : float(M_PI) / 8.0f;
+    Position const& pos = { x, y, z, 0.0f };
```

Comment:
But what if we could use the function in SAI or for default mob spawn. Then we need kinda xyz.